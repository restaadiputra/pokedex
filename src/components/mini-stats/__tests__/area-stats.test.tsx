import React from 'react';
import { render, screen } from '@testing-library/react';
import AreaStats from '../area-stats';

test('should render the area with capitalize on first letter', () => {
  render(<AreaStats area="kanto" />);
  expect(screen.getByText('Kanto')).toBeInTheDocument();
  expect(screen.getByText('Kanto')).toMatchSnapshot();
});
