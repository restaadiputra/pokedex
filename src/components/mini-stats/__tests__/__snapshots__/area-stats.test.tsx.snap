// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the area with capitalize on first letter 1`] = `
<p
  class="text-white tracking-wide text-sm opacity-75 font-light"
>
  <span
    class="font-normal"
  >
    Region : 
  </span>
  Kanto
</p>
`;
